ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -49,6 +57,11 @@
 except NameError:
     OAUTH2_PROVIDER = {}
 
+try:
+    CACHES  # noqa: F821
+except NameError:
+    CACHES = {}
```

Comment:
Need to change this, I guess if CACHES is not defined at all, django is happy.  If it's defined but empty (or not containing a 'default' entry) then it isn't.  So if not defined, don't mess with it.